assets/css/lesson.scss

```diff
@@ -145,6 +145,14 @@ img {
 // Miscellaneous.
 //----------------------------------------
 
+article.figures p {
+  text-align: center;
```

Comment:
I would suggest to remove this centering because this way figure titles of different lengths will be start at different locations. Without centering the positions of figure titles is more consistent.